tailwind.config.js

```diff
@@ -7,6 +7,13 @@ const tailwindGradients = require('tailwindcss-gradients')
 module.exports = {
   purge: ['./src/**/*.tsx', './src/**/*.css'],
   theme: {
+    screens: {
+      sm: '640px',
+      md: '768px',
+      lg: '1024px',
+      xl: '1280px',
+      xxl: '1400px',
+    },
```

Comment:
I think you could do this under `extend` and only have to specify the new breakpoint of `xxl`